src/lazy/encoder/binary/v1_1/value_writer.rs

```diff
@@ -2920,21 +3238,23 @@ mod tests {
         Ok(())
     }
 
+    // TODO: Revisit this unit test, we need to change the system macro being called.. originally
+    // it was calling (:none), but with the parameter tracking it will fail since :none does not
```

Comment:
`meta` is like `none`, but it takes arguments. You could use `meta` and leave the rest of this test largely unchanged, I think.

Reply:
Oh nice, good call. I'll still have to circle back around to it though after writing e-groups in binary is implemented.